grid-proxy/internal/gpuindexer/gpuindexer.go

```diff
@@ -29,6 +30,7 @@ type NodeGPUIndexer struct {
 	batchSize              int
 	nodesGPUResultsChan    chan []types.NodeGPU
 	nodesGPUBatchesChan    chan []types.NodeGPU
+	newNodeTwinIDChan      chan []uint32
```

Comment:
Why is that a channel of int32 slice? this should keep receiving the latest twinID inserted on the nodes table,  no?

Reply:
ah forget it checks the nodes between the last id you checked and the last id on the table, could be more than one indeed